tools/emterpretify.py

```diff
@@ -480,7 +480,8 @@ def handle_async_post_call():
 CASES[ROPCODES['INTCALL']] = '''
     lz = HEAPU8[(HEAP32[pc + 4 >> 2] | 0) + 1 | 0] | 0; // FUNC inst, see definition above; we read params here
     ly = 0;
-    assert(((EMTSTACKTOP + 8|0) <= (EMT_STACK_MAX|0))|0); // for return value
+    if (((EMTSTACKTOP + 8|0) > (EMT_STACK_MAX|0))|0) // for return value
+      abortStackOverflowEmterpreter();
```

Comment:
I realized there's a problem here. We remove assertions if ASSERTIONS is not on, but this PR adds a check that is not removed automatically - so it adds overhead to non-ASSERTIONS builds.

Instead, it would be good to only add the test if ASSERTIONS (which is defined in tools/emterpretify.py)